packages/dialtone-vue2/components/popover/popover.vue

```diff
@@ -879,7 +879,7 @@ export default {
 
     focusInitialElement () {
       if (this.initialFocusElement === 'dialog') {
-        this.$refs.content.$el.focus();
+        this.$refs.content?.$el.focus();
```

Comment:
Should we put it on $el too to be safe 

`this.$refs.content?.$el?.focus();`

Reply:
Fixed in [34a01f6](https://github.com/dialpad/dialtone/pull/165/commits/34a01f69cc069d42f25953994df8443f5699f3a0)